Library/Formula/wine.rb

```diff
@@ -61,13 +61,6 @@ def require_universal_deps?
     sha256 "30766403f5064a115f61de8cacba1defddffe2dd898b59557956400470adc699"
   end
 
-  # Patch to fix MSI creation issues.
```

Comment:
Rather than removing this patch completely, which will whack stable users who still need it, it should be moved into the `stable do` block